doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -835,27 +802,72 @@ Inductive definitions
 
 We gradually explain simple inductive types, simple annotated inductive
 types, simple parametric inductive types, mutually inductive types. We
-explain also co-inductive types.
+also explain co-inductive types.
 
 Simple inductive types
 ~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Inductive @ident : {? @sort } := {? | } @ident : @type {* | @ident : @type }
+.. cmd:: Inductive @inductive_definition {* with @inductive_definition }
+
+   .. insertprodn inductive_definition record_binder_body
+
+   .. prodn::
+      inductive_definition ::= {? > } @ident_decl {* @binder } {? : @term } {? := {? @constructor_list_or_record_decl } } {? @decl_notations }
+      constructor_list_or_record_decl ::= %| {+| @constructor }
+      | @ident @constructor_type %| {*| @constructor }
+      | @ident @constructor_type
+      | @ident %{ {+; @record_field } %}
+      | %{ {+; @record_field } %}
+      constructor_type ::= {* @binder } {? @of_type }
+      record_field ::= {* #[ {*, @attr } ] } @record_binder {? %| @num } {? @decl_notations }
+      record_binder ::= @name
+      | @name @record_binder_body
+      record_binder_body ::= {* @binder } @of_type
```

Comment:
`record_body` doesn't seem right, either.  It's a subpart of the definition of a single `record_field`.  Maybe it could be `field_body`?  That still echos `def_body`.

There's also already a definition for `record_body` in the gallina extensions chapter that this conflicts with.  I'd rather not pull that apart now since there may be rippling changes from removing that.

Reply:
I made it `field_body` for now.  At least that way CI will generate HTML output.

Reply:
Great!

Reply:
> why records do not support the `Eval` construct

No particular reasons. Nobody was feeling it was missing probably, plus, maybe, the fact that the use of a capital letter for `Eval` does not look very nice visually.

If you want to implement it, it'll have to be passed from `g_vernac.mlg` to `Record.definition_structure` via `vernacentries.ml` with a call to `red_constant_body` after interpretation, following the model done in `ComDefinition.interp_definition`.